checks/raw/shell_download_validate.go

```diff
@@ -1005,9 +1005,14 @@ func collectUnpinnedPackageManagerDownload(startLine, endLine uint, node syntax.
 				},
 				Pinned: asBoolPointer(!isNugetUnpinned(c)),
 				Type:   checker.DependencyUseTypeNugetCommand,
+				Remediation: &finding.Remediation{
+					Text: "pin your dependecies by either enabling central package management " +
+						"(https://learn.microsoft.com/nuget/consume-packages/Central-Package-Management) " +
+						"or using a lockfile (https://learn.microsoft.com/nuget/consume-packages/" +
+						"package-references-in-project-files#locking-dependencies)",
+				},
```

Comment:
This should only be added if the dependency is unpinned right? So something like:
```go
if isNuget(c) {
                pinned := !isNugetUnpinned(c)
                var remediation *finding.Remediation
                if !pinned {
                       remediation = &finding.Remediation{
			   Text: // omitted, but same as before
		      }
                 }
		r.Dependencies = append(r.Dependencies,
			checker.Dependency{
				Location: &checker.File{
					Path:      pathfn,
					Type:      finding.FileTypeSource,
					Offset:    startLine,
					EndOffset: endLine,
					Snippet:   cmd,
				},
				Pinned: &pinned,
				Type:   checker.DependencyUseTypeNugetCommand,
				Remediation: remediation,
```